jenkins_wrapper_cookbook/recipes/default.rb

```diff
@@ -86,7 +97,7 @@
 
 execute 'install_plugins' do
   command <<-EOH.gsub(/^ {4}/,'')
-  source /etc/profile
+  . /etc/profile
```

Comment:
Because the shell interpreter isn't defined it defaults to `/bin/sh` which on Ubuntu/Debian is `/bin/dash` which doesn't support the Bash-ism `source`. The `.` version of `source` works on any POSIX system.